Godeps/_workspace/src/k8s.io/kubernetes/pkg/client/unversioned/helper.go

```diff
@@ -580,7 +580,7 @@ func DefaultKubernetesUserAgent() string {
 	version := version.Get().GitVersion
 	seg := strings.SplitN(version, "-", 2)
 	version = seg[0]
-	return fmt.Sprintf("%s/%s (%s/%s) kubernetes/%s", path.Base(os.Args[0]), version, gruntime.GOOS, gruntime.GOARCH, commit)
+	return fmt.Sprintf("%s/%s (%s/%s) openshift/%s", path.Base(os.Args[0]), version, gruntime.GOOS, gruntime.GOARCH, commit)
```

Comment:
> origin vs openshift vs aep?

"openshift" is hardcoded in https://github.com/openshift/origin/blob/master/pkg/client/client.go#L315 which this now matches.